### PR TITLE
Add TVN domains for RWS

### PR DESCRIPTION
**Pull request by dzmarzlak**

### CODE REVIEW

**Review by Airborn22 — APPROVED** [review]
LGTM, merging

**Review by shuranhuang — APPROVED** [review]
LGTM. Merge.

**Review by Hamedpahlan — COMMENTED** [review]
Rewawe

**Comment by Hamedpahlan** [comment]
afbd39f0bde250d06c0c9d12385a50329088f8a5

**Comment by CrabbyJim** [comment]
Have many problems with the program, it will not allow me to login to your
site. It also logs me out of several other places and takes forever to
start a function.
I would like to clear Gmail account ***@***.*** to use this as the
only option for download to the phone for the other application. All data
for social networks should use the best email address for all and a
firewall built to isolate shadowtrack to the above address 😀. If a 🔑 is
built to gain access to all other programs, they may be accessed from guest
browser on Chrome book.

**Comment by CrabbyJim** [comment]
Thank you again my friends!
<?> IS THE PACKAGE READY FOR DEPOSIT ON 5/14 <?>

File: related_website_sets.JSON [hunk 1]
```diff
@@ -639,7 +639,19 @@
...
+      "associatedSites": [
+            "https://tvn24.pl", 
+            "https://zdrowietvn.pl"
+      ],
+      "rationaleBySite": {
+          "https://zdrowietvn.pl": "Educational service that is owned by TVN S.A.. Information about the connection with TVN is included in the footer of this website",
+          "https://tvn24.pl": "News service that is owned by TVN S.A.. Information about the connection with TVN is included in the footer of this website"
+      }
     }
-  }
```
Comment (Parasos) [inline related_website_sets.JSON:643]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Comment by ishfaqsiyal** [comment]
Yes

**Comment by bosspab** [comment]
https://www.facebook.com/ManilaPlayOfficial/videos/1180276779904554/?app=fbl

**Review by PYRTUS1997 — APPROVED** [review]

**Pull request merged**